components/artist_page_cta/view.coffee

```diff
@@ -53,10 +53,18 @@ module.exports = class ArtistPageCTAView extends Backbone.View
       afterAuthPath: @afterAuthPath
     @$(".artist-page-cta-overlay__register input[name='name']").focus()
     @$('.artist-page-cta-overlay__close').on 'click', @closeOverlay
+    @disableScroll()
+
+  disableScroll: ->
+    $('#main-layout-container').addClass('artist-page-cta__no-scroll')
```

Comment:
Yea I BEM'ed it like this to make it obvious that the special behavior is coming from this component's CSS.

It behaves like you'd expect, there's no more scrollbar/scrolling once the overlay is up.